README.md

```diff
@@ -1,3 +1,11 @@
 cppcheclipse is an Eclipse plugin which integrates [cppcheck](http://sourceforge.net/projects/cppcheck/) with the [CDT project](https://eclipse.org/cdt/). You can run/configure cppcheck from the Eclipse UI.
 
+To build the project by hand it requires maven and Linux platform (maven under Windows will not compile the project properly). To compile run following commands:
```

Comment:
To build the project by hand -> rather "To build the project on the command line"

Reply:
Why doesn't it work on Windows?

Reply:
Because your unit tests will fail because you use paths like /test/testfolder/testfile